From 981dca64afe60e5732cdb3e39889e4c8b7dc02ca Mon Sep 17 00:00:00 2001
Message-Id: <981dca64afe60e5732cdb3e39889e4c8b7dc02ca.1698332284.git.gerbach@fs-net.de>
From: Philipp Gerbach <gerbach@fs-net.de>
Date: Thu, 26 Oct 2023 16:57:53 +0200
Subject: [PATCH] Fix runtime library pathes

The beep_regestry and aiur_regestry should not use the sysroot path in
for the runtime libraries
---
 plugins/aiurdemux/meson.build | 2 +-
 plugins/beepdec/meson.build   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/aiurdemux/meson.build b/plugins/aiurdemux/meson.build
index b09c657..4d56f5c 100644
--- a/plugins/aiurdemux/meson.build
+++ b/plugins/aiurdemux/meson.build
@@ -14,7 +14,7 @@ gstaiurdemux = library('gstaiurdemux',
   install_dir : plugins_install_dir,
 )
 
-imxparser_libdir = imx_parser_dep.get_pkgconfig_variable('libdir')
+imxparser_libdir = '/usr/lib/imx-mm/parser/'
 
 aiur_registry = configuration_data()
 aiur_registry_install_dir = '@0@/share'.format(get_option('prefix'))
diff --git a/plugins/beepdec/meson.build b/plugins/beepdec/meson.build
index 147f689..fac004f 100644
--- a/plugins/beepdec/meson.build
+++ b/plugins/beepdec/meson.build
@@ -17,7 +17,7 @@ gstbeepdec = library('gstbeepdec',
   install_dir : plugins_install_dir,
 )
 
-acodec_libdir = fslaudiocodec_dep.get_pkgconfig_variable('libdir')
+acodec_libdir = '/usr/lib/imx-mm/audio-codec/'
 
 beep_registry = configuration_data()
 beep_registry_install_dir = '@0@/share'.format(get_option('prefix'))
-- 
2.40.1

